dev/make-single-file-spm

```diff
@@ -51,7 +51,7 @@ function setup_swiftpm_package() {
 
     cd "$destination"
     cat > Package.swift <<"EOF"
-// swift-tools-version:5.1
+// swift-tools-version:5.2
```

Comment:
Did you validate this script works? I suspect it now produces a broken SPM Package manifest.

Reply:
we also don't need this change. You can still depend on packages that have newer requirements with older manifests.

Reply:
Good point